Example/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+load("@build_bazel_rules_ios//rules:app.bzl", "ios_application")
```

Comment:
Please nest the example under a subdirectory specificallyPlease nest the example under a subdirectory specifically about mixed-language frameworks